### PR TITLE
Ran tests for Django 5.0, updated tests to include new area-invalid t…

### PR DESCRIPTION
**Pull request by mixtah**
### Summary
Wanted to use it for Django >5.0
Tests failed for Django >5.0, but was just a slight change in the returned HTML injected by Django in an invalid form and not an apparent breaking change.

Updated the test to reflect this change in Django >5.0 while keeping the test valid for Django <5.0 without the aria-invalid field. 

Generally tested this version in my project and appears to work well, at least doesn't crash anything (yet). 

I'd give it the "It works on my machine" quality control ranking.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,5 +27,5 @@ Changes made in this fork:
 - Upgraded to use pyproject.toml
 - Use github actions over travis-ci
 - Python 3.12 compatibilty
-- Django 4.2 compatibilty
```
Comment (SunnyR) [inline README.md:30]: Let's change this to include constraints as follows:

```Django >4.2,<5.1 compatibility```

File: pyproject.toml [hunk 2]
```diff
@@ -39,7 +39,7 @@ exclude = [
 
 [tool.poetry.dependencies]
 python = "~3.12"
-django = "^4.2"
```
Comment (SunnyR) [inline pyproject.toml:42]: Similar to the above comment, we should limit this to what we can support - `Django >4.2,<5.1 compatibility`

**Review by SunnyR — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mixtah) [reply README.md:30]: Done

File: pyproject.toml (hunk 2 above)
Comment (mixtah) [reply pyproject.toml:42]: Done

**Review by SunnyR — APPROVED** [review]

**Pull request merged**